### PR TITLE
feat: Update guide for applying and using keys in camel

### LINKED ISSUE #950: [Feature Request] A unify api key management for tools
### Required prerequisites

- [X] I have searched the [Issue Tracker](https://github.com/camel-ai/camel/issues) and [Discussions](https://github.com/camel-ai/camel/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [ ] Consider asking first in a [Discussion](https://github.com/camel-ai/camel/discussions/new).

### Motivation

It is difficult for users to create all api keys for tools. We can have a unified api key for users to use all available tools in CAMEL.

This feature maybe some research and design.

### Solution

_No response_

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by NeilJohnson0930**
## Description

Update guide for applying and using keys in camel.

## Motivation and Context

`close #950 ` 

- [ ] I have raised an issue to propose this change ([required](https://github.com/camel-ai/camel/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)
- [ ] Example (update in the folder of example)

## Implemented Tasks

- [ ] Subtask 1
- [ ] Subtask 2
- [ ] Subtask 3

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] I have read the [CONTRIBUTION](https://github.com/camel-ai/camel/blob/master/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: api_key
Comment (lightaime) [inline api_key:1]: Maybe make it as a `.env` file with all the variable names and empty as default values. Turn the current description into comments so people can use `load_dotenv()` to load environment variables. Also tell how people can use `load_dotenv()` in comments.

File: README.md [hunk 1]
```diff
@@ -248,6 +248,9 @@ Please note that the environment variable is session-specific. If you open a new
 
 After setting the OpenAI API key, you can run the `role_playing.py` script. Find tasks for various assistant-user roles [here](https://drive.google.com/file/d/194PPaSTBR07m-PzjS-Ty6KlPLdFIPQDd/view?usp=share_link).
 
+For more details about the key names in project and how to apply key, 
+you can refer to [here](https://github.com/camel-ai/camel/api_key).
```
Comment (lightaime) [inline README.md:252]: Move to `https://github.com/camel-ai/camel`
```suggestion
you can refer to [here](https://github.com/camel-ai/camel/.env).
```

File: api_key
Comment (lightaime) [inline api_key:1]: Let's devide the keys by modules like `models`, `toolkits`. Do not need to add numbers.

**Review by lightaime — COMMENTED** [review]
Thanks @NeilJohnson0930! It looks great!! Add some comments.

**Review by Wendong-Fan — APPROVED** [review]
Thanks @NeilJohnson0930 ! Added one more commit here

**Pull request merged**